src/docs/user/references/VariablesReference.adoc

```diff
@@ -208,6 +208,41 @@ ProgressFile.setProgress(variables.get("PA_TASK_PROGRESS_FILE"), 50);
 | <<_selection, selection>>
 | -
 
+| *Scheduler Rest URL*. The REST url of the scheduler server. The public REST url is also available (if the setting pa.scheduler.rest.public.url is defined in `PROACTIVE_HOME/config/scheduler/settings.ini`)
+| variables.get( "PA_SCHEDULER_REST_URL" ) or variables.get( "PA_SCHEDULER_REST_PUBLIC_URL" )
+| $variables_PA_SCHEDULER_REST_URL or $variables_PA_SCHEDULER_REST_PUBLIC_URL
+| ${PA_SCHEDULER_REST_URL} or ${PA_SCHEDULER_REST_PUBLIC_URL}
+| all scripts
+| -
+
+| *Catalog Rest URL*. The REST url of the catalog service. The public REST url is also available (if the setting pa.catalog.rest.public.url is defined in `PROACTIVE_HOME/config/scheduler/settings.ini`)
+| variables.get( "PA_CATALOG_REST_URL" ) or variables.get( "PA_CATALOG_REST_PUBLIC_URL" )
+| $variables_PA_CATALOG_REST_URL or $variables_PA_CATALOG_REST_PUBLIC_URL
+| ${PA_CATALOG_REST_URL} or ${PA_CATALOG_REST_PUBLIC_URL}
+| all scripts
+| -
+
+| *Cloud Automation Rest URL*. The REST url of the cloud automation service. The public REST url is also available (if the setting pa.cloud-automation.rest.public.url is defined in `PROACTIVE_HOME/config/scheduler/settings.ini`)
+| variables.get( "PA_CLOUD_AUTOMATION_REST_URL" ) or variables.get( "PA_CLOUD_AUTOMATION_REST_PUBLIC_URL" )
+| $variables_PA_CLOUD_AUTOMATION_REST_URL or $variables_PA_CLOUD_AUTOMATION_REST_PUBLIC_URL
+| ${PA_CLOUD_AUTOMATION_REST_URL} or ${PA_CLOUD_AUTOMATION_REST_PUBLIC_URL}
+| all scripts
+| -
+
+| *Job Planner Rest URL*. The REST url of the job-planner service. The public REST url is also available (if the setting pa.job-planner.rest.public.url is defined in `PROACTIVE_HOME/config/scheduler/settings.ini`)
+| variables.get( "PA_JOB_PLANNER_REST_URL" ) or variables.get( "PA_JOB_PLANNER_REST_PUBLIC_URL" )
+| $variables_PA_JOB_PLANNER_REST_URL or $variables_PA_JOB_PLANNER_REST_PUBLIC_URL
+| ${PA_JOB_PLANNER_REST_URL} or ${PA_JOB_PLANNER_REST_PUBLIC_URL}
+| all scripts
+| -
+
+| *Notification Service Rest URL*. The REST url of the notification service. The public REST url is also available (if the setting pa.notification-service.rest.public.url is defined in `PROACTIVE_HOME/config/scheduler/settings.ini`)
```

Comment:
**pa.notification-service.rest.public.url** (better to put it in italic to differentiate it from the rest of the text)